Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "PushNotifications",
+        "repositoryURL": "https://github.com/pusher/push-notifications-swift.git",
+        "state": {
+          "branch": null,
+          "revision": "7f074a47beab45ca020806c06c88e65c80a7227e",
+          "version": "3.0.2"
```

Comment:
Should we be on v 3.0.2 of push-notifications-swift?

Reply:
👍

Reply:
This file is automatically generated by Xcode. It can be updated to `3.0.3` it doesn't have to as Xcode will resolve to the most recent version anyway (this file is only important for command line dependency resolution). `3.0.3` adds a fix for Objective-C which we don't support.